lighthouse-core/config/default.js

```diff
@@ -632,41 +666,41 @@ module.exports = {
       "name": "Accessibility",
       "description": "These checks highlight opportunities to [improve the accessibility of your app](https://developers.google.com/web/fundamentals/accessibility).",
       "audits": [
-        {"id": "accesskeys", "weight": 1},
-        {"id": "aria-allowed-attr", "weight": 1},
-        {"id": "aria-required-attr", "weight": 1},
-        {"id": "aria-required-children", "weight": 1},
-        {"id": "aria-required-parent", "weight": 1},
-        {"id": "aria-roles", "weight": 1},
-        {"id": "aria-valid-attr-value", "weight": 1},
-        {"id": "aria-valid-attr", "weight": 1},
-        {"id": "audio-caption", "weight": 1},
-        {"id": "button-name", "weight": 1},
-        {"id": "bypass", "weight": 1},
-        {"id": "color-contrast", "weight": 1},
-        {"id": "definition-list", "weight": 1},
-        {"id": "dlitem", "weight": 1},
-        {"id": "document-title", "weight": 1},
-        {"id": "duplicate-id", "weight": 1},
-        {"id": "frame-title", "weight": 1},
-        {"id": "html-has-lang", "weight": 1},
-        {"id": "html-lang-valid", "weight": 1},
-        {"id": "image-alt", "weight": 1},
-        {"id": "input-image-alt", "weight": 1},
-        {"id": "label", "weight": 1},
-        {"id": "layout-table", "weight": 1},
-        {"id": "link-name", "weight": 1},
-        {"id": "list", "weight": 1},
-        {"id": "listitem", "weight": 1},
-        {"id": "meta-refresh", "weight": 1},
-        {"id": "meta-viewport", "weight": 1},
-        {"id": "object-alt", "weight": 1},
-        {"id": "tabindex", "weight": 1},
-        {"id": "td-headers-attr", "weight": 1},
-        {"id": "th-has-data-cells", "weight": 1},
-        {"id": "valid-lang", "weight": 1},
-        {"id": "video-caption", "weight": 1},
-        {"id": "video-description", "weight": 1},
+        {"id": "accesskeys", "weight": 1, "tags": ["a11y-correct-attributes"]},
```

Comment:
> since we're only supporting the first entry, should we just drop the array?

accessibility only uses one, but forcing one already seems bad when we just planned last week to achieve the PWA goals with multiple tags on a one